### ISSUE TITLE
[Sharing] Guest users cannot expose a port

### ISSUE THREAD

**Issue by svenefftinge** [gitpod-io/gitpod#1241]
### Describe the bug
Guest users on shared workspaces cannot expose a port.
The pop up is shown, but pressing expose doesn't do anything.

Workaround:

Enter the port in a .gitpod.yml and reload the browser.

> Going to be obsolete with next deployment, as users as well as guest users will automatically have access to all ports.

**Comment by ivanraybuglosa** [comment]
I actually did the workaround but still can't show the running app. Is there anything else that I can do? Thanks for the help!

**Comment by svenefftinge** [comment]
I tried that myself before posting it here and it worked for me. To avoid missunderstandings here is what I did:
1) created a `.gitpod.yml` in the root of the project with the following contents
```
ports:
 - port: 4000
```
2) shared the workspace with someone. The person needs to be logged in with gitpod.
3) start a webserver, e.g. `python3 -m http.server 4000`

The guest user should the a notification, asking whether to preview or open in browser.
